recipes/cfchecker/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "2.0.5" %}
+
+package:
+    name: cfchecker
+    version: {{ version }}
+
+source:
+    fn: cfchecker-{{ version }}ceda.p1.tar.gz
+    url: https://pypi.python.org/packages/source/c/cfchecker/cfchecker-{{ version }}ceda.p1.tar.gz
+    sha1: 8709708c8c9417e91403a0e0aa539d3bae55f346
+    patches:
+        # Ensure our udunits2 lib is used on all platforms.
+        - ctypes.patch
+
+build:
+    skip: True  # [win or py3k]
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
+    entry_points:
+        - cfchecks = cfchecker:cfchecks_main
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - numpy 1.8.*  # Needs cdat_lite that needs oldnumeric.
```

Comment:
Same thoughts as above. Not that we need to do anything with this recipe per se, but we should probably have our own version of `numpy` 1.8.